src/main/java/io/kestra/plugin/klaviyo/tasks/CreateCampaign.java

```diff
@@ -0,0 +1,534 @@
+package io.kestra.plugin.klaviyo.tasks;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.kestra.core.exceptions.IllegalVariableEvaluationException;
+import io.kestra.core.http.HttpRequest;
+import io.kestra.core.http.HttpResponse;
+import io.kestra.core.http.client.HttpClient;
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.property.Property;
+import io.kestra.core.models.tasks.RunnableTask;
+import io.kestra.core.models.tasks.Task;
+import io.kestra.core.runners.RunContext;
+import io.kestra.plugin.klaviyo.constants.Constants;
+import io.kestra.plugin.klaviyo.models.requests.CreateCampaignRequest;
+import io.kestra.plugin.klaviyo.models.responses.error.ErrorResponse;
+import io.kestra.plugin.klaviyo.models.responses.success.CampaignCreateSuccessResponse;
+import io.kestra.plugin.klaviyo.utils.Utils;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+import org.apache.hc.core5.http.ClassicHttpRequest;
+import org.slf4j.Logger;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "Create a Klaviyo campaign",
+    description = "Creates a new campaign in Klaviyo with message, audiences, and tracking options."
+)
+@Plugin(
+    examples = {
+        @Example(
+            title = "Create a simple campaign",
+            code = {
+                "privateApiKey: \"<KLAVIYO_API_KEY>\"",
+                "revision: \"2025-07-15\"",
+                "name: \"September Promo Campaign\"",
+                "sendStrategy:",
+                "  method: \"immediate\"",
+                "sendOptions:",
+                "  use_smart_sending: true"
+            }
+        )
+    }
+)
+public class CreateCampaign extends Task implements RunnableTask<CreateCampaign.Output> {
+    @Schema(
+        title = "Klaviyo Private API Key",
+        description = "Klaviyo Private API Key"
```

Comment:
```suggestion
        description = "Pass a private API Key as a secret or KV pair. Follow [the documentation](https://help.klaviyo.com/hc/en-us/articles/7423954176283) to create your API key."
```
if `description` = `title`, you know that description is incomplete :)